cmd/minikube/cmd/config/profile.go

```diff
@@ -17,20 +17,27 @@ limitations under the License.
 package config
 
 import (
+	"os"
+	
 	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
 	pkgConfig "k8s.io/minikube/pkg/minikube/config"
 	"k8s.io/minikube/pkg/minikube/console"
 	"k8s.io/minikube/pkg/minikube/exit"
 )
 
 // ProfileCmd represents the profile command
 var ProfileCmd = &cobra.Command{
-	Use:   "profile MINIKUBE_PROFILE_NAME.  You can return to the default minikube profile by running `minikube profile default`",
-	Short: "Profile sets the current minikube profile",
-	Long:  "profile sets the current minikube profile.  This is used to run and manage multiple minikube instance.  You can return to the default minikube profile by running `minikube profile default`",
+	Use:   "profile [MINIKUBE_PROFILE_NAME].  You can return to the default minikube profile by running `minikube profile default`",
+	Short: "Profile gets or sets the current minikube profile",
+	Long:  "profile sets the current minikube profile, or gets the current profile if no arguments are provided.  This is used to run and manage multiple minikube instance.  You can return to the default minikube profile by running `minikube profile default`",
 	Run: func(cmd *cobra.Command, args []string) {
-		if len(args) != 1 {
-			exit.Usage("usage: minikube profile MINIKUBE_PROFILE_NAME")
+		if len(args) == 0 {
```

Comment:
hey, not sure if others agree, but I think the code would read better like:
```
  if len(args) == 0 { 
    profile := viper.GetString(pkgConfig.MachineProfile)
    console.OutLn(profile)
    os.Exit(0)
  }

  if len(args) > 1 { 
    exit.Usage("usage: minikube profile [MINIKUBE_PROFILE_NAME]")
  }
```

@MPLew-is WDYT?

Reply:
That's perfectly fine by me, I was just preserving as much of the existing code as possible. I'll make the change and push.